requirements-dev.txt

```diff
@@ -0,0 +1,2 @@
+-r requirements.txt
```

Comment:
This line is a reference to `requirements.txt`. It means that whenever you install the development dependencies, you also install the regular dependencies, since you need them for development too.